CHANGES.rst

```diff
@@ -559,6 +559,9 @@ astropy.uncertainty
 astropy.units
 ^^^^^^^^^^^^^
 
+- Ensured that output from test functions of and comparisons between quantities
```

Comment:
```suggestion
- Ensure that output from test functions of and comparisons between quantities
```

Reply:
Hmm, I meant this as "[I made a bug fix that ] ensured that....". But this is definitely fine too, so I'll just commit your suggestion and merge.